resources/shared/test-invoker.mjs

```diff
@@ -24,7 +24,24 @@ export class TimerTestInvoker extends TestInvoker {
     }
 }
 
-export class RAFTestInvoker extends TestInvoker {
+class AsyncTimerTestInvoker extends TestInvoker {
```

Comment:
Can we instead just delete `TimerTestInvoker` as we discussed in a separate PR?

Reply:
We probably want to land the removal PR first.

Reply:
thanks for opening https://github.com/WebKit/Speedometer/pull/466